.vscode/settings.json

```diff
@@ -23,5 +23,6 @@
 
     // Always use project's provided typescript compiler version
     "typescript.tsdk": "node_modules/typescript/lib",
+    "tslint.configFile": "tslint.json",
```

Comment:
Please revert this change. It might cause problems when unsing different tslint.json files in different directories in the future.

Reply:
Will remove in the next update